package.json

```diff
@@ -35,7 +35,6 @@
     "@ffprobe-installer/ffprobe": "1.0.9",
     "@ngx-translate/core": "11.0.1",
     "angular2-virtual-scroll": "0.3.1",
```

Comment:
please remove the `,` from the end of this line - 👍 else I get "_Failed to parse json_" on `npm start`

Reply:
Ah! From merging! Thanks! 👍